javascript/new-src/src/factories/ola.js

```diff
@@ -63,6 +76,9 @@ ola.factory('$ola', ['$http', '$window', 'OLA',
           strip = false;
         }
       }
+      while (integers.length < highestChannelNumberUsed) {
+        integers.push(OLA.MIN_CHANNEL_VALUE);
+      }
```

Comment:
I think you could drop this and tweak the strip stuff, which was trying to do the same thing (introduced in #874) but had this bug.

Reply:
Good point. I will simplify the logic as soon as possible.

Reply:
done

Reply:
Thanks.